e2e/app/sidebar/runtime-panel.ts

```diff
@@ -374,31 +374,23 @@ export default class RuntimePanel extends BaseSidebar {
   }
 
   /**
-   * Open Runtime sidebar, check status for running.
+   * Open Runtime sidebar, wait for running or error status.
```

Comment:
Logically, why would `waitForRunning` return when it's in the error state? Consider renaming the method or else updating the semantics some other way